packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -139,6 +128,7 @@ globalThis.__dangerous_ON_edge_converter_returns_request = true;
     );
   }
 
+  console.log();
```

Comment:
It's this intended?

Reply:
Yes, for this empty line:
![Screenshot 2024-12-18 at 11 36 06](https://github.com/user-attachments/assets/86afa29c-01c0-45e5-aaed-5c6850d80027)

probably a `\n` in the other console.log would be better/clearer?

Reply:
Moving this log to l222 inside `patchCodeWithValidations` as "Applied N patches" might be a bit clearer, up to you.

Reply:
sounds good, I don't have a strong preference here 🙂